src/main/java/sggw/wzim/czasnawypad/exception/ApplicationExceptions.java

```diff
@@ -0,0 +1,28 @@
+package sggw.wzim.czasnawypad.exception;
+
+public class ApplicationExceptions {
+
+    public static class AttractionNotFoundException extends RuntimeException {
+        public AttractionNotFoundException(String message) {
+            super(message);
+        }
+    }
+
+    public static class UserNotFoundException extends RuntimeException {
+        public UserNotFoundException(String message) {
+            super(message);
+        }
+    }
+
+    public static class RatingNotFoundException extends RuntimeException {
+        public RatingNotFoundException(String message) {
+            super(message);
+        }
+    }
+
+    public static class NotYourRatingException extends RuntimeException {
```

Comment:
```suggestion
    public static class RatingAccessDeniedException extends RuntimeException {
```
Is this Exception used anywhere?